src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageResponseCommentUiTest.java

```diff
@@ -188,4 +195,68 @@ private InstructorFeedbackResultsPage loginToInstructorFeedbackResultsPage(Strin
                 .withSessionName(testData.feedbackSessions.get(fsName).getFeedbackSessionName());
         return loginAdminToPage(resultsUrl, InstructorFeedbackResultsPage.class);
     }
+
+    private void addEmptyCommentToResponseAndCheckStatusMessage(String commentModelId) {
+        // TODO: `clickCommentModalButton` should wait for modal to open before returning
```

Comment:
Don't add TODO here. Create issue if needed.

Reply:
Use `waitForModalShown`?

Reply:
I haven't added it. It was added in https://github.com/TEAMMATES/teammates/pull/8859/files, that got merge few days back. I have only refactored it.

Reply:
Shall I open new issue for these TODOs ?

Reply:
@tshradheya Can you create issues for these TODOs? Unless you will fix them soon, we should not add more TODOs in our code base. It's hard to track and manage them.